docs/content/en/docs/topics/provision-a-cluster/_index.md

```diff
@@ -14,123 +20,176 @@ description: Steps to provision a Cassandra cluster in Kubernetes
+  ----     ------            ----                  ----               -------
+  Warning  FailedScheduling  3m22s (x51 over 73m)  default-scheduler  0/6 nodes are available: 6 node(s) didn't match pod affinity/anti-affinity, 6 node(s) didn't satisfy existing pods anti-affinity rules.
+```
+
+The output reveals a `FailedScheduling` event.
+
+To resolve the mismatch between the configured size and the available nodes, consider the following option to set the `allowMultipleNodesPerWorker` property to relax the constraint of only allowing one Cassandra pod per Kubernetes worker node.
+
+Here is an updated k8ssandra-values.yaml with `allowMultipleNodesPerWorker`:
 
 ```yaml
-.
-.
-.
-                   "description": "Total sizes of the data on distinct nodes",
-                   "description": "Maximum JVM Heap Memory size (worst node) and minimum available heap size",
-  size: 3
+cassandra:
+  clusterName: my-k8ssandra
+  allowMultipleWorkersPerNode: true
+  # resources must be set when allowMultipleWorkersPerNode is true.   
+  resources: 
+    requests:
+      cpu: 2
+      memory: 2Gi
+    limits:
+      cpu: 2
+      memory: 2Gi
+  # It is not required to set the heap but is recommended.
+  heap:
+    size: 1024M
+    newGenSize: 512M
+  datacenters:
+  - name: dc1
+    size: 3
 ```
 
-### Scale down the cluster
+When applied to the test cluster, this configuration updates the size property of the `CassandraDatacenter`. Then cass-operator will in turn update the underlying `StatefulSet`.
 
-Similarly, to scale down, lower the current `size` to conserve cloud resource costs, if the new value can support your computing requirements in Kubernetes. For example, this time we'll lower the size to 1, and again set the CassandraDatacenter name `dc1` (currently required each time) with the command:
+If you check the status of the `CassandraDatacenter` object, there should be a `ScalingUp` condition with its status set to `true`. It should look like this:
 
 ```bash
-helm upgrade k8ssandra k8ssandra/k8ssandra --set cassandra.datacenters\[0\].size=1,cassandra.datacenters\[0\].name=dc1
+ kubectl get cassandradatacenter dc1 -o yaml
 ```
 
-**Output**:
+**Output:**
 
 ```bash
-Release "k8ssandra" has been upgraded. Happy Helming!
-NAME: k8ssandra
-LAST DEPLOYED: Thu Mar  4 21:14:05 2021
-NAMESPACE: default
-STATUS: deployed
-REVISION: 3
+...
+status:
+  cassandraOperatorProgress: Updating
+  conditions:
+  - lastTransitionTime: "2021-03-30T22:01:48Z"
+    message: ""
+    reason: ""
+    status: "True"
+    type: ScalingUp
+...
 ```
 
-Again, verify the upgrade:
+After the new nodes are up and running, `nodetool cleanup` should run on all of the nodes except the new ones to remove keys and data that no longer belong to those nodes. There is no need to do this manually. The cass-operator product, which again is installed with K8ssandra, automatically runs `nodetool cleanup` for you.
```

Comment:
Similar to above I would replace `product` with `Deployment`.

Reply:
Inc'd your comments with subsequent commit.